tokio/Cargo.toml

```diff
@@ -59,6 +59,7 @@ net = [
   "windows-sys/Win32_System_SystemServices",
 ]
 process = [
+  "rt",
```

Comment:
Since you have a fallback, you could remove this by checking for both `linux` and the `rt` feature. If both aren't set, fallback to the signal based approach.

Reply:
Yeah that makes sense, will change that after work.

Reply:
Thanks, I've reverted this change and change `cfg` for `PidfdReaper` to `#[cfg(all(target_os = "linux", feature = "rt"))]`